src/Index.js

```diff
@@ -745,6 +773,10 @@ Index.prototype.searchRules = function(params, callback) {
   });
 };
 
+Index.prototype.exportRules = function(hitsPerPage) {
```

Comment:
Need to pass callback down the line too no? Otherwise here it does not supports callback form.

Reply:
I was thinking to have this as a "new functions don't have callbacks anymore", but probably better to keep it indeed. Problem is that we use promises internally for this, so you'll still need support (maybe okay with the polyfill we have)